decidim-blogs/app/views/decidim/blogs/posts/_actions.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="blog__actions">
+  <div class="blog__actions-left">
+    <% if endorsements_enabled? %>
+      <%# REDESIGN_PENDING: replace toggler %>
+      <button id="endorsers_list-trigger" class="cursor-pointer" data-toggler=".is-active" data-toggle="endorsers_list-trigger endorsers_list" role="button">
```

Comment:
I understand that this feature is not completed yet as per the previous discussion, but just to list this issue here, I also noticed that the endorsement button is not working correctly in this situation:
- Go to a blog post you have not yet endorsed
- Endorse the blog post
- Refresh the page
- Remove the endorsement from the blog post
- Now you still see your face in the endorsers "toggle" unless you reload the page

I would expect my face to disappear from the toggle element when I unendorse the post.

The problem also exists the other way around: when you first endorse the post, your face does not appear in the toggle without a reload of the page.

Reply:
Addressed to https://github.com/decidim/decidim/issues/9671

Reply:
Could you remove the `role="button"` on the button element because when using the `<button>` element, it already indicates its correct role.